docs/en/Storage.md

```diff
@@ -1,132 +1,261 @@
+    These are guidelines, not an exact science - pick what sounds best now and run with it.  The best choice for a complicated usage is non-obvious and often takes hands-on experience, so just trying something will help.  For most situations both options work well even if they're not perfect, and remember that data can always be copied later if you change your mind.
 
 ## Disks
 
-Disks are the familiar hard drive (or SSD) style file systems! You use them
-directly in Kubeflow when you add workspace and data volumes to your notebook
-server. They are automatically mounted at the directory you choose, and serve as
-a reliable way to preserve your data. Even if you delete your server later, you
-can still remount your disks to a new one – by default they are never destroyed.
+### Overview
+
+Disks are the familiar SSD style file systems! When creating your notebook
+server, you request them by adding Data Volumes to your notebook server. They
+are automatically mounted at the directory you choose, and serve as a simple and
+reliable way to preserve data attached to a Notebook Server. Even if you delete
+your Notebook Server later, your disk will not be deleted. You can then mount
+that same disk (with all its contents) to a new Notebook Server later as shown:
 
 ![Adding an existing volume to a new notebook server](images/kubeflow_existing_volume.png)
 
-This is a super simple way to store your data. And if you share a workspace with
-a team, your whole team can use the same server's disk just like a shared drive.
+Or, if you're done with the disk, [delete it](#deleting-disk-storage).
+
+<!-- prettier-ignore -->
+??? info "Disks are never destroyed by default, but you pay for them whether they're attached to a Notebook Server or not"
+    When you create a disk, you agree to [pay](#pricing) for the cost of that drive until it is destroyed, just like how you agree to pay for the CPU/RAM you request when making a Notebook Server.  Unlike with Notebook Server CPU/RAM, however, when you shut down a Notebook Server your disks are not deleted.  This is to let you reuse those drives, but means that when you shutdown a Notebook Server **you are still paying for the storage**.  You pay for any disk listed in the Notebook Volumes section of the Notebook Server page (above).  Storage is [pretty cheap](#pricing) but can still add up - see [Deleting Disk Storage](#deleting-disk-storage) for more info.
+    
+As disks can be attached to a Notebook Server and reused, a typical usage pattern could be:
+
+- At 9AM, create a Notebook Server (request 2CPU/8GB RAM and a 32GB attached
+  disk)
+- Do work throughout the day, saving results to the attached disk
+- At 5PM, shut down your Notebook Server to avoid paying for it overnight
+  - NOTE: The attached disk **is not destroyed** by this action
+- At 9AM the next day, create a new Notebook Server and **attach your existing
+  disk**
+- Continue your work...
```

Comment:
This is good, but I would move it into the **Pricing** section, maybe under a `??? tip` or a subsubsection for "Tips for reducing cost".

Also I would try to shrink the content of the "Disks are never destroyed" bubble. Something like

You pay for the cost of data volumes until you manually destroy them, however the storage is pretty inexpensive. Estimate the cost of 100gb to be (**INSERT DOLLAR AMOUNT HERE**). See Deleting Disk Storage for more info.